internal/patroni/reconcile_test.go

```diff
@@ -219,7 +285,41 @@ volumes:
           path: ~postgres-operator/patroni.ca-roots
         - key: patroni.crt-combined
           path: ~postgres-operator/patroni.crt+key
-	`))
+	`,
+		},
+	}
+	for name, tt := range tests {
+		t.Run(name, func(t *testing.T) {
+			cluster := new(v1beta1.PostgresCluster)
+			err := cluster.Default(context.Background(), nil)
+			assert.NilError(t, err)
+			cluster.Name = "some-such"
```

Comment:
Do we have any agreement on how to name variables in tests, or does it not matter in this case?

Reply:
this is upstream code that was just moved to a different place. if you notice the diff you will see where it was before.